src/Rings/FreeAssociativeAlgebraIdeal.jl

```diff
@@ -184,37 +200,90 @@ Ideal generating system with elements
 function groebner_basis(I::FreeAssociativeAlgebraIdeal,
   deg_bound::Int=-1;
   protocol::Bool=false,
-  interreduce::Bool=false)
+  interreduce::Bool=false,
+  algorithm::Symbol=:default,
+  probabilistic::Bool = false
+  )
   isdefined(I, :gb) && (I.deg_bound == -1 || I.deg_bound >= deg_bound) && return I.gb
-  I.gb = groebner_basis(IdealGens(gens(I)), deg_bound; ordering=:deglex, protocol=protocol, interreduce=interreduce)
+  I.gb = groebner_basis(IdealGens(gens(I)), deg_bound; ordering=:deglex, protocol=protocol, interreduce=interreduce, algorithm=algorithm)
   I.deg_bound = deg_bound
   return I.gb
 end
 function groebner_basis(g::IdealGens{<:FreeAssociativeAlgebraElem},
   deg_bound::Int=-1;
   ordering::Symbol=:deglex,
   protocol::Bool=false,
-  interreduce::Bool=false)
-  gb = groebner_basis(collect(g), deg_bound; ordering=ordering, protocol=protocol, interreduce=interreduce)
+  interreduce::Bool=false,
+  algorithm::Symbol=:default,
+  probabilistic::Bool = false
+  )
+  gb = groebner_basis(collect(g), deg_bound; ordering=ordering, protocol=protocol, interreduce=interreduce, algorithm=algorithm)
```

Comment:
```suggestion
  gb = groebner_basis(collect(g), deg_bound; ordering=ordering, protocol=protocol, interreduce=interreduce, algorithm=algorithm, probabilistic=probabilistic)
```
doesn't this need to get forwarded?